### PR TITLE
Docs: Changing wrapper topic titles

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR:
- Changes wrapper topic titles

[skip changelog]

### CODE REVIEW

**Comment by kirszenbaum** [comment]
@AMBudnik - would you be able to take over the review on this one? Thank you!

File: docs/10.0/guides/integrate-with-react/react-redux-example.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: 'React: Redux example'
-metaTitle: 'React: Redux example - Guide - Handsontable Documentation'
+title: 'Redux example'
```
Comment (AMBudnik) [inline docs/10.0/guides/integrate-with-react/react-redux-example.md:2]: Hey @kirszenbaum I see that `React: Redux example` is replaced by `Redux` example. I assume that it did not switch to `Redux example in React` as Redux is used only in React so it's not needed to add that note.

**Review by AMBudnik — APPROVED** [review]
👍  all is good. Thanks for fixing that `hotColumn` terminology as well.

File: docs/10.0/guides/integrate-with-react/react-redux-example.md (hunk 1 above)
Comment (AMBudnik) [reply docs/10.0/guides/integrate-with-react/react-redux-example.md:2]: nvm, I saw that you did the same with Vuex so it answers by question

**Pull request merged**